README.md

```diff
@@ -1,11 +1,14 @@
 # ActiveRecord Cloud Spanner Adapter
 
-# 🚨THIS CODE IS STILL UNDER DEVELOPMENT🚨
-
 [Google Cloud Spanner](https://cloud.google.com/spanner) adapter for ActiveRecord.
 
 ![rubocop](https://github.com/googleapis/ruby-spanner-activerecord/workflows/rubocop/badge.svg)
 
+This project provides a Cloud Spanner adapter for ActiveRecord. It has the __Preview__ release status and supports ActiveRecord 6.0.x/6.1.x with Ruby 2.6 or higher.
```

Comment:
nit: we actually support Ruby 2.5 and higher, but I don't know if that is something we want to communicate.

(See https://github.com/googleapis/ruby-spanner-activerecord/blob/dd6f707a37c5f76741f0a293e8a1e9568b63c8c9/.github/workflows/nightly-unit-tests.yaml#L13)

Reply:
As you mentioned, 2.5 is eol. So it probably makes future maintenance easier without including it here.